app/views/admin/tracks/index.html.haml

```diff
@@ -11,27 +11,52 @@
         %th Name
         %th Short name
         %th Description
+        %th Submitter
         %th Color
+        %th State
+        %th Included in the Cfp
         %th Actions
       %tbody
         - @tracks.each do |track|
           %tr
             %td
-              = link_to(admin_conference_program_track_path(@conference.short_title, track.short_name)) do
+              = link_to(admin_conference_program_track_path(@conference.short_title, track)) do
                 = track.name
             %td
               = track.short_name
             %td
               %p
                 = truncate(track.description)
             %td
+              - if track.self_organized?
+                = link_to track.submitter.name, admin_user_path(track.submitter)
+              - else
+                N/A
+            %td
               %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
                 = track.color
             %td
+              - if track.self_organized?
+                = track.state
+              - else
+                N/A
+            %td
+              - if track.self_organized?
+                = check_box_tag "#{@conference.short_title}_#{track.short_name}", track.id, track.cfp_active,
+                  class: 'switch-checkbox', method: :patch,
+                  url: toggle_cfp_inclusion_admin_conference_program_track_path(@conference.short_title, id: track.short_name)+"?included=",
+                  data: { size: 'small',
+                          on_color: 'success',
+                          off_color: 'warning',
+                          on_text: 'Yes',
+                          off_text: 'No' }
+              - else
+                %i.fa.fa-check
+            %td
               .btn-group{role: "group"}
-                = link_to 'Edit', edit_admin_conference_program_track_path(@conference.short_title, track.short_name),
+                = link_to 'Edit', edit_admin_conference_program_track_path(@conference.short_title, track),
                 method: :get, class: 'btn btn-primary'
-                = link_to 'Delete', admin_conference_program_track_path(@conference.short_title, track.short_name),
+                = link_to 'Delete', admin_conference_program_track_path(@conference.short_title, track),
```

Comment:
For self organized tracks, destroy button should be disabled, so that the record doe snot get deleted, only moved to withdrawn/cancelled state.